modules/standard/Map.chpl

```diff
@@ -74,6 +73,7 @@ module Map {
 
     /* If `true`, this map will perform parallel safe operations. */
     param parSafe = false;
+    param chpl_nilable = true;
```

Comment:
Just for understanding - why `chpl_nilable` in this case instead of `_isNilable`? Just to indicate it's not supposed to be user facing? Not really important, but it might help me in the future.

Reply:
I think we want to avoid using leading underscores in general. Yes, it was meant to mark it as internal/private.